Rn entry 2022.09.0 (#3584)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/release-notes/2022/index.md

title: Okta API Products release notes 2022
---

## September

### Monthly release 2022.09.0

| Change | Expected in Preview Orgs |
|--------------------------------------------------------------------------|--------------------------|

> | [Non-deletable default authorization server is in Early Access (EA)](#non-deletable-default-authorization-server-is-in-early-access-ea) | August 31, 2022 |

Reviewer: ```suggestion
| [Non-deletable default authorization server is EA in Preview](#non-deletable-default-authorization-server-is-ea-in-preview) | August 31, 2022 |
```

> | [PKCE validation for OIDC app integrations is in General Availability (GA)](#pkce-validation-for-oidc-app-integrations-is-ga-in-production) | July 7, 2022 |

Reviewer: ```suggestion
| [PKCE validation for OIDC app integrations is GA in Production](#pkce-validation-for-oidc-app-integrations-is-ga-in-production) | July 7, 2022 |
```

| [Signed request support for generic SAML IdP is GA in Production](#signed-request-support-for-generic-saml-idp-is-ga-in-production) | July 7, 2022 |
| [Step-up authentication using ACR values is EA in Preview](#step-up-authentication-using-acr-values-is-ea-in-preview) | August 31, |

> | [API for suppressing email notifications is in General Availability](#api-for-suppressing-email-notifications-is-in-general-availability) | May 11, 2022 |

Reviewer: ```suggestion
| [API for suppressing email notifications is GA](#api-for-suppressing-email-notifications-is-in-general-availability) | May 11, 2022 |
```

| [Access Denied error message customization is GA in Preview](#access-denied-error-message-customization-is-ga-in-preview) | August 31, 2022 |
| [Dynamic IdP routing is GA in Preview](#dynamic-idp-routing-is-ga-in-preview) | June 8, 2022 |
| [The update logo for application operation is updated](#the-update-logo-for-application-operation-is-updated) | August 31, 2022 |
| [Password restriction for shared SWA app accounts is GA in Production](#password-restriction-for-shared-swa-app-accounts-is-ga-in-production) | July 7, 2022 |
| [Developer documentation updates in 2022.09.0](#developer-documentation-updates-in-2022090) | August 31, 2022 |
| [Bugs fixed in 2022.09.0](#bugs-fixed-in-2022090) | August 31, 2022|

> #### Non-deletable default authorization server is in Early Access (EA)

Reviewer: ```suggestion
#### Non-deletable default authorization server is EA in Preview
```

#### Signed request support for generic SAML IdP is GA in Production

Using signed SAML requests ensures that incoming requests are from genuine applications. When this is configured, Okta only accepts SAML requests signed using the certificate associated with the app integration. Having signed SAML requests also resolves scenarios where the Assertion Consumer Service (ACS) URL requested after authentication can be one of several domains or URLs. When a Service Provider sends a signed authentication request, Okta can accept dynamic ACS values as part of the SAML request and posts the SAML assertion response to the ACS value specified in the request. See the Advanced Settings section of [Create SAML app integrations using AIW](https://help.okta.com/okta_help.htm?type=oie&id=csh-apps-aiw-saml).<!--OKTA-525803-->

#### Step-up authentication using ACR values is EA in Preview

> Users want seamless access to certain resources, but organizations want to increase the users' level of assurance before they access anything sensitive. It’s difficult to strike a balance between implementing stronger security controls and offering a frictionless experience for your users to easily interact with an application. Okta now supports the `acr_values` parameter, which refers to Authentication Context Class Reference. Each value defines a specific set of assurance level requirements that the protected resource requires from the authentication event associated with the access and ID tokens. See [Step-up authentication using ACR values]().<!--OKTA-525790-->

Reviewer: don't forget to insert the link to the doc

#### API for suppressing email notifications is in General Availability

> This API allows you to change who receives email notifications for each individual email template. You can suppress them completely, or send them to admins only. This unlocks testing scenarios which warrant using production user directories, and prevents end users from getting test emails. It also allows extensibility for customers who would like to use a third party email sender through Hooks or Workflows. See [Email template settings](/docs/reference/api/brands/#email-template-settings).<!--OKTA-526580-->

Reviewer: ```suggestion
This API allows you to change who receives email notifications for each individual email template. You can suppress them completely or send them to admins only. This unlocks testing scenarios that warrant using production user directories and prevents end users from getting test emails. It also allows extensibility for customers who would like to use a third party email sender through Hooks or Workflows. See [Email template settings](/docs/reference/api/brands/#email-template-settings).<!--OKTA-526580-->
```
